part-overview.tex

```diff
@@ -221,13 +219,13 @@ \section{System requirements}
 \label{sec-ledgersmb-system-requirements}
 
 The INSTALL file which comes with every LedgerSMB software release should be
-considered the authorative source of system requirements. In summary, the
+considered the canonical source for system requirements. In summary, the
 following technical components are required:
 
 \begin{itemize}
 \item The Apache web server (others may or may not work)
-\item Perl 5.10 or higher, with additional modules
-\item PostgreSQL 8.3 or higher, with contribs
+\item Perl 5.32 or higher, with additional modules

```

Comment:
1.9 requires 5.24, 1.10 wants 5.32 (but then again, 1.10 wants a higher Pg version)

Reply:
Keeping the book in sync with version numbers seems like a nightmare.  It would be better to reference something on-line that is maintained and remove all of these version numbers.  I found https://ledgersmb.org/content/system-requirements and that seems like it would work.

How about I generalize the requirements (no version numbers) then add a link to [this](https://ledgersmb.org/content/system-requirements) with some explanation?

Reply:
Yup. Very good idea. The fewer the number of tables with similar info, the better, because less work to maintain.